src/main/java/reactor/netty/channel/BootstrapHandlers.java

```diff
@@ -487,6 +489,22 @@ public static Bootstrap removeMetricsSupport(Bootstrap b) {
 		return b;
 	}
 
+	public static Bootstrap updateResolverForProxySupport(Bootstrap b) {
```

Comment:
this will need some javadoc

Reply:
Added Javadoc.  Please let me know if you would like more details.